### DISCUSSION TITLE
add ok google command and make it show results from google and youtube

### DISCUSSION THREAD

**Discussion by ghost** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#466] [Ideas]
yup

**Comment by im-coder-lg** [comment]
Check this:

- [x] Ok Google/Hey Google(has external issue, made an issue: #472)

Results from Google and YouTube has problems because it **used** to work but the Google Assistant SDK was changed by Google themselves so it doesn't work.

**Comment by ghost** [comment]
then maybe use youtube data api v3 and google search api?

**Comment by im-coder-lg** [comment]
Don't need to, because I found a solution for this. Go to my gist profile and hit the GAUDC Personal Results markdown file. Steps are listed there. Do that, it worked for me. Got to add the screenshot soon.

**Reply by farhanfHARAHAP** [reply]
Where the hell is the SCREENSHOOOT!?

**Reply by im-coder-lg** [reply]
Yo, I added it! Wait, let me add a photo to show that I did add it.

**Reply by im-coder-lg** [reply]
![Screenshot_20210922_075900.jpg](https://user-images.githubusercontent.com/76845820/134274068-8a42e7b0-4d79-401a-984b-2935058a92e8.jpg)
Fine? I did, it has to be your issue that you can't see the screenshot.
